Error while checking build/O2/o2-dataflow for e1b2a564b8a631b38a6b0f25cfa45622b2c56cd5 at 2022-09-14 14:42:
```
## sw/BUILD/O2-latest/log
100% tests passed, 0 tests failed out of 427
 85/106 Test  #99: test_Framework_test_ParallelProducer ....................***Failed    2.83 sec
[ERROR] pid 24752 (merger) crashed with 1
[ERROR] SEVERE: Device merger (24752) returned with 1
99% tests passed, 1 tests failed out of 101
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/9816/e1b2a564b8a631b38a6b0f25cfa45622b2c56cd5/build_O2_o2-dataflow/pretty.html).

--- a/Detectors/Calibration/workflow/src/MeanVertexCalibratorSpec.cxx
+++ b/Detectors/Calibration/workflow/src/MeanVertexCalibratorSpec.cxx
@@ -40,7 +40,7 @@ void MeanVertexCalibDevice::init(InitContext& ic)
   float rangeZ = params->rangeZ;
   int nSlots4SMA = params->nSlots4SMA;
   auto slotL = params->tfPerSlot;
-  auto delay = params->maxTFdelay;
+  float delay = (float)params->maxTFdelay / slotL; // in timeslot calibration, the delay is in fraction of timeslot length
   auto nPointsForSlope = params->nPointsForSlope;
   mCalibrator = std::make_unique<o2::calibration::MeanVertexCalibrator>(minEnt, nbX, rangeX, nbY, rangeY, nbZ, rangeZ, nSlots4SMA, nPointsForSlope);
   mCalibrator->setSlotLength(slotL);

--- a/Detectors/Calibration/workflow/src/MeanVertexCalibratorSpec.cxx
+++ b/Detectors/Calibration/workflow/src/MeanVertexCalibratorSpec.cxx
@@ -40,7 +40,7 @@ void MeanVertexCalibDevice::init(InitContext& ic)
   float rangeZ = params->rangeZ;
   int nSlots4SMA = params->nSlots4SMA;
   auto slotL = params->tfPerSlot;
-  float delay = (float)params->maxTFdelay / slotL; // in timeslot calibration, the delay is in fraction of timeslot length
+  float delay = float(params->maxTFdelay) / slotL; // in timeslot calibration, the delay is in fraction of timeslot length
   auto nPointsForSlope = params->nPointsForSlope;
   mCalibrator = std::make_unique<o2::calibration::MeanVertexCalibrator>(minEnt, nbX, rangeX, nbY, rangeY, nbZ, rangeZ, nSlots4SMA, nPointsForSlope);
   mCalibrator->setSlotLength(slotL);

--- a/prodtests/full-system-test/aggregator-workflow.sh
+++ b/prodtests/full-system-test/aggregator-workflow.sh
@@ -148,7 +148,9 @@ fi
 if [[ $AGGREGATOR_TASKS == BARREL_TF ]] || [[ $AGGREGATOR_TASKS == ALL ]]; then
   # PrimVertex
   if [[ $CALIB_PRIMVTX_MEANVTX == 1 ]]; then
-    add_W o2-calibration-mean-vertex-calibration-workflow "" "MeanVertexCalib.tfPerSlot=55000"
+    if [[ -z $TFPERSLOTS_MEANVTX ]]; then TFPERSLOTS_MEANVTX=55000; fi
+    DELAYINTFS_MEANVTX="10"
+    add_W o2-calibration-mean-vertex-calibration-workflow "" "MeanVertexCalib.tfPerSlot=$TFPERSLOTS_MEANVTX;MeanVertexCalib.maxTFdelay=$DELAYINTFS_MEANVTX"
   fi
 
   # TOF

